packages/flutter_tools/templates/ffi_plugin/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.10)
+set(PROJECT_NAME "{{projectName}}")
+project(${PROJECT_NAME} LANGUAGES CXX)
+
+# Invoke the build for native code shared with the other target platforms.
+include(${CMAKE_CURRENT_SOURCE_DIR}/../src/CMakeLists.txt)
+
+# List of absolute paths to libraries that should be bundled with the plugin.
+set({{projectName}}_bundled_libraries
+  $<TARGET_FILE:{{projectName}}>
```

Comment:
It's not-obvious that this is defined in the other CMake file; I would explicitly comment this.